README.md

```diff
@@ -1,13 +1,5 @@
-This is on master
+Branch-2 yo
 
-dogpatch edit
+Branch-2 content!
```

Comment:
Start a review comment - let's leave it a minute this time

Edit start a review comment

https://github.com/yamiacat/MoreTestRenamed/pull/32